static/rss.xml

```diff
@@ -4,7 +4,7 @@
         <description><![CDATA[RSS feed for QuantStack website blog page]]></description>
         <link>https://quantstack.net</link>
         <generator>RSS for Node</generator>
-        <lastBuildDate>Tue, 01 Jul 2025 16:14:17 GMT</lastBuildDate>
+        <lastBuildDate>Tue, 01 Jul 2025 17:54:10 GMT</lastBuildDate>
```

Comment:
For the record, is this file generated? If so, perhaps use https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github to mask it in diffs?

Reply:
Yes. You are right. That's something I have in my list of things to fix.